services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -207,7 +208,9 @@ def get_dynamic_sidecar_spec(
                 "Labels": {},
                 "Mounts": mounts,
             },
-            "Placement": {"Constraints": []},
+            "Placement": {
+                "Constraints": [app_settings.DIRECTOR_V2_SERVICES_CUSTOM_CONSTRAINTS]
```

Comment:
THOUGHTS:
What would happen if ``DIRECTOR_V2_SERVICES_CUSTOM_CONSTRAINTS="foo;alksdf      alksdf===>>??"`` 
What about security? Can I inject here a string with a command that makes the system vulnerable?

Reply:
This is just a [service placement constraint](https://docs.docker.com/engine/swarm/services/#control-service-placement). From what I can tel it's a rule that does string matching. So I would not worry about it. Usually you have something like `node.labels.region==east`.